test/interval_tests/construct_tests.jl

```diff
@@ -52,36 +49,43 @@ facts("Constructing intervals") do
     @fact convert(Interval, 0.1) --> Interval(0.09999999999999999, 0.1)
     @fact convert(Interval, BigFloat(0.1)) --> Interval(big(0.1))
     a = @interval(0.1)
-    @fact convert(Interval{Rational{Int}},a) --> Interval(1//10)
-    @fact convert(Interval{Rational{BigInt}},pi) --> Interval{Rational{BigInt}}(pi)
+
+    #@fact convert(Interval{Rational{Int}},a) --> Interval(1//10)
+    #@fact convert(Interval{Rational{BigInt}},pi) --> Interval{Rational{BigInt}}(pi)
 
 
     # Constructors from the macros @interval, @floatinterval @biginterval
     set_interval_precision(53)
+
     a = @interval(0.1)
     b = @interval(pi)
+
     @fact nextfloat(a.lo) --> a.hi
     @fact typeof(a) --> Interval{BigFloat}
     @fact a --> @biginterval("0.1")
-    @fact float(a) --> @floatinterval(0.1)
```

Comment:
Since `float{Interval}` exists, we may keep this test as well, even if it does the same as the next line.